### PR TITLE
Extend the tutorial to describe location of decrypted secrets

### PR DESCRIPTION
**Pull request by edrex**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,8 +161,11 @@ but, if you want to (change the system based on your system):
    ```nix
    age.secrets.secret1.file = ../secrets/secret1.age;
    ```
+      
 5. NixOS rebuild or use your deployment tool like usual.
 
+   The secret will be decrypted to `age.secrets.secret1.path` (`/run/secrets/secret1` by default). For per-secret options controlling ownership etc, see [modules/age.nix](modules/age.nix).
```
Comment (asymmetric) [inline README.md:167]:
```suggestion
   The secret will be decrypted to the value of `age.secrets.secret1.path` (`/run/secrets/secret1` by default). For per-secret options controlling ownership etc, see [modules/age.nix](modules/age.nix).
```

**Review by asymmetric — APPROVED** [review]
Nit

**Comment by happysalada** [comment]
Just want to bump this PR up. I think it's an improvement!

**Review by ryantm — APPROVED** [review]

**Comment by ryantm** [comment]
Thanks and sorry for the delay.

**Pull request merged**